examples/vortex-mpi.py

```diff
@@ -108,6 +127,26 @@ def main(ctx_factory=cl.create_some_context):
     nodes = thaw(actx, discr.nodes())
     current_state = initializer(nodes)
 
+    vis_timer = None
+
+    if logmgr:
+        logmgr_add_device_name(logmgr, queue)
+        logmgr_add_default_discretization_quantities(logmgr, discr, dim)
+
+        logmgr.add_watches(["step.max", "t_step.max", "t_log.max",
+                            "min_temperature", "norm_momentum1"])
```

Comment:
Can these be changed easily to temperature, pressure (min, max) for both?

Reply:
Yes, you would need to call `logmgr.add_watches(["min_temperature", "max_temperature", "min_pressure", "max_pressure"])`

Reply:
Doing that would provide a more relevant example, but I realize that it would then not be demonstrative of the loggers facility for handling vector quantities.